conda/pprof/build.sh

```diff
@@ -2,10 +2,14 @@
 set -ex
 export GOPATH="$PREFIX/pprof"
 # Build/Install pprof from google at specified hash
-go install github.com/google/pprof@c488b8fa1db3fa467bf30beb5a1d6f4f10bb1b87
+go install github.com/google/pprof@latest
+
+ls -latrh ${GOPATH}/pprof/bin
+ls -latrh ${GOPATH}/pprof/pkg
```

Comment:
```suggestion
ls -latrh ${GOPATH}/bin
ls -latrh ${GOPATH}/pkg
```

Looking at the linux output, I think you need to remove the pprof dir in these lines, as it should be under `GOPATH`